src/Illuminate/Contracts/Log/Channel.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Illuminate\Contracts\Log;
+
+use Psr\Log\LoggerInterface;
+use Monolog\Formatter\FormatterInterface;
+use Illuminate\Contracts\Events\Dispatcher;
+
+interface Channel extends LoggerInterface
+{
+    /**
+     * Prepare the channel for logging.
+     *
+     * @param  array  $options
+     * @return void
+     */
+    public function prepare(array $options = []);
+
+    /**
+     * Set the formatter to be used on the log channel.
+     *
+     * @param  \Monolog\Formatter\FormatterInterface  $formatter
+     * @return void
+     */
+    public function setFormatter(FormatterInterface $formatter);
+
+    /**
+     * Setup an event listener for automatic logging.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $events
+     * @param  string  $event
+     * @param  string  $level
```

Comment:
See my comment below, could reference`\Psr\Log\LogLevel`

Reply:
Nope. It'd still be a string. `LogLevel::DEBUG` is a string